Building cmasher

```
___________________________________________ Test_import_cmaps.test_CMasher_cmaps[__pycache__] ___________________________________________

self = <tests.test_utils.Test_import_cmaps object at 0x7f222279a900>, cm_name = '__pycache__'

    @pytest.mark.parametrize("cm_name", cm_names)
    def test_CMasher_cmaps(self, cm_name):
        # Check if provided cm_name is registered in CMasher and MPL
        for name in (cm_name, cm_name + "_r"):
>           cmr_cmap = getattr(cmr, name)
E           AttributeError: module 'cmasher' has no attribute '__pycache__'. Did you mean: '__cached__'?

/home/USERNAME/.cache/yay/python-cmasher/src/CMasher-1.9.0/tests/test_utils.py:358: AttributeError
======================================================== short test summary info ========================================================
FAILED tests/test_utils.py::Test_import_cmaps::test_CMasher_cmaps[__pycache__] - AttributeError: module 'cmasher' has no attribute '__pycache__'. Did you mean: '__cached__'?
```

--- a/tests/test_utils.py
+++ b/tests/test_utils.py
@@ -58,7 +58,9 @@ COLORMAPS_DIR = CMASHER_DIR / "colormaps"
 # As all colormaps have their own directories, save them instead
 _IGNORED = ["PROJECTS", "__pycache__"]
 cm_names = [
-    p.name for p in sorted(COLORMAPS_DIR.glob("*")) if p.is_dir() and p not in _IGNORED
+    p.name
+    for p in sorted(COLORMAPS_DIR.glob("*"))
+    if p.is_dir() and p.name not in _IGNORED
 ]
 
 # Obtain list of all colormaps registered in MPL

